gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -77,6 +77,12 @@ def add_arguments(self, parser):
               'large number of files are specified by input_pattern. '
               'Note that each VCF file must still contain valid header files '
               'even if this is provided.'))
+    parser.add_argument(
+        '--force_merge_header_conflicts',
+        type='bool', default=False, nargs='?', const=True,
+        help=('If true, all the resolvable conflicts (e.g, Integer vs Float)'
+              'in in the header fields of the given VCF files are resolved.'
+              'If not set, such conflicts raise errors.'))
```

Comment:
If type conversions do not affect data integrity, we might want to always do these conversions. I would consider defaulting the parameter to True or even removing it altogether to avoid the added complexity.

Reply:
That's a good point. I am not sure why Asha suggested having a flag for this. Should we keep it as is and change it later once Asha is back?

Reply:
Keeping it as I think is okay for now. Maybe make a note or TODO or github issue of it to address it after break.

Reply:
Done! PTAL.